lib/tests/test_conflicts.rs

```diff
@@ -539,25 +542,29 @@ line 5
         @r###"
     Some(
         [
-            Resolved(
-                "line 1\n",
-            ),
-            Conflict(
-                Conflict {
-                    removes: [
-                        "line 2\nline 3\nline 4\n",
-                        "line 2\nline 3\nline 4\n",
-                    ],
-                    adds: [
-                        "line 2\nleft\nline 4\n",
-                        "right\n",
-                        "line 2\nforward\nline 3\nline 4\n",
-                    ],
-                },
-            ),
-            Resolved(
-                "line 5\n",
-            ),
+            Conflict {
+                removes: [],
+                adds: [
+                    "line 1\n",
+                ],
+            },
+            Conflict {
+                removes: [
+                    "line 2\nline 3\nline 4\n",
+                    "line 2\nline 3\nline 4\n",
+                ],
+                adds: [
+                    "line 2\nleft\nline 4\n",
+                    "right\n",
+                    "line 2\nforward\nline 3\nline 4\n",
+                ],
+            },
+            Conflict {
```

Comment:
We could consider changing the Debug implementation for `Conflict` to print `Conflict::resolved(...)` here.

This wouldn't be necessary if Conflict were an enum. I'm not sure which is a better idea, but I agree that if we make something an enum with a Resolved option, it should be `Conflict`

Reply:
Yes, that might be nice, but I'll leave it as is for now. I'll like to try making `Conflict` an enum instead, but I'll probably not do that either yet :)